src/components/navbar/index.cjsx

```diff
@@ -93,6 +94,7 @@ module.exports = React.createClass
             title={<UserName/>}
             ref='navDropDown'>
             {menuItems}
+            <AdminConsole/>
```

Comment:
`CurrentUserStore.isAdmin()` is undefined when `render` is called.  It only gets updated later.  So the admin link is never shown.